azure-pipelines-PR.yml

```diff
@@ -0,0 +1,137 @@
+# Branches that trigger builds on PR
```

Comment:
I assume this is a copy. So have not reviewed the PR yml.

Reply:
yes